kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/ApplyMojo.java

```diff
@@ -163,16 +131,6 @@
     @Parameter(property = "jkube.deploy.ignoreRunningOAuthClients", defaultValue = "true")
     private boolean ignoreRunningOAuthClients;
 
-    /**
-     * Should we create external Ingress/Routes for any LoadBalancer Services which don't already have them.
-     * <p>
-     * We now do not do this by default and defer this to the
-     * <a href="https://github.com/jkubeio/exposecontroller/">exposecontroller</a> to decide
-     * if Ingress or Router is being used or whether we should use LoadBalancer or NodePorts for single node clusters
-     */
-    @Parameter(property = "jkube.deploy.createExternalUrls", defaultValue = "false")
-    private boolean createExternalUrls;
```

Comment:
Is this parameter still supported via properties?

Reply:
I have moved this to resource mojo. This flag enables `Ingress` generation